src/components/BusinessCard.vue

```diff
@@ -33,23 +32,31 @@
         </div>
         <div class="text-xl font-bold mb-1 pb-1 border-b">{{business.name}}</div>
       </header>
-      <div class="text-sm">
-        <div v-if="expanded">{{business.long_description}}</div>
+      <div class="text-sm flex-grow">
+        <div v-if="expanded">{{business.long_description || business.short_description}}</div>
         <div v-else>{{business.short_description}}</div>
       </div>
-      <div style="flex: 1">
+      <div class="flex flex-row items-end">
+        <div class="text-sm flex flex-row align-baseline flex-grow">
+          <span
+            v-for="tag in tags"
+            :key="tag.id"
+            class="box-border whitespace-no-wrap text-gray-500 pr-2 flex-end"
+            :class="`tag-${tag.name}`"
```

Comment:
Funny how these can be separate but I think you could also just combine them

Reply:
Combined!